Extending/Backoffice-Search/index.md

```diff
@@ -0,0 +1,84 @@
+---
+keywords: Backoffice Search
+versionFrom: 8.6.0
+meta.Title: "Backoffice Search"
+meta.Description: "A guide to customization of Backoffice Search"
+---
+
+# Backoffice Search
+
+Backoffice search is usefull tool, which support you in finding your content. Umbraco by default will allow you to search using only few fields: 
+
+| Node Type    | Propagated Fields    |
+| ------------ | -------------------- |
+| All Nodes    | UmbracoFileFieldName |
+| Media Nodes  | File path            |
+| Member Nodes | email, loginName     |
+|              |                      |
+
+Umbraco 8.6 introduce new easy way for extend/override that fields, which is called IUmbracoTreeSearcherFields. This service exposed hardcoded internal list of fields and allow for modification on them.
+
+### Adding custom properties to backoffice search
+
+#### All Node types
+
+```
+public class CustomInternalSearchConstants : InternalSearchConstants,     IInternalSearchConstants
```

Comment:
InternalSearchConstants and IInternalSearchConstants, have been renamed to be UmbracoTreeSearcherFields and IUmbracoTreeSearcherFields

Reply:
true, forgot to update my example code, will upgrade PR soon